README.md

```diff
@@ -26,9 +14,7 @@ cd ..
 python3 -m venv env
 source env/bin/activate
 python -m pip install --upgrade pip
-pip install -r mne-python/requirements.txt
 pip install -e mne-python
-pip install -r requirements.txt
+pip install -r ./q1k_eeget_init/requirements.txt
 pip install -e .
-python -m ipykernel install --user --name q1k_init_kernel
```

Comment:
I have only ever seen --user installs cause hard-to-debug issues with python so I don't think we should suggest this. Using a virtual env removes the need to use the --user option anyway (there are no users in a venv). Removed.